src/pulsar_client.erl

```diff
@@ -310,9 +339,7 @@ do_try_connect([URI | Servers], Opts0, Res) ->
                             {ok, {Sock, Opts}};
                         {{error, CommandError}, <<>>} ->
                             do_try_connect(Servers, Opts, Res#{{Host, Port} => CommandError})
-                    end;
-                OtherMsg ->
-                    do_try_connect(Servers, Opts, Res#{{Host, Port} => {unexpected_msg, OtherMsg}})
```

Comment:
Removed this clause because it was grabbing the `ping` from `start_reconnect_timer`...  Maybe this should be a `gen_statem`?